tests/Plugins/GitHub_EndToEndTest.py

```diff
@@ -169,6 +169,18 @@ def test_DefaultBranchValue(self, pat_args, snapshot):
         assert result.exit_code == -1, result.output
         assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
 
+    # ----------------------------------------------------------------------
+    def test_NoDescription(self, pat_args, snapshot):
+        """Test if no repository description is available."""
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-Description-allow-empty", "false"],

```

Comment:
We shouldn't add a `true` or `false` value with a flag - the flag itself is sufficient to toggle the default.